### DISCUSSION TITLE
anyone know what this means?

### DISCUSSION THREAD

**Discussion by K0-oki3** [victornpb/undiscord#568] [4. Bugs and Issues]
i cant delete any messages at all, all it says is this
![Screenshot 2023-12-09 155907](https://github.com/victornpb/undiscord/assets/153398153/c188127e-db29-4ada-b1e2-11b79c298708)
:

**Comment by Sieglinde-V** [comment]
Check all the tabs on the left and make sure your autofill isn't filling in your email address into one of the boxes.

**Reply by nowayyback** [reply]
O meu vai até uns 25 e para do nada, oque pode ser?
